Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-03
Initial Package Version: 5.50
Origin: NA
Description: Fixes compilation against system zlib.

$LastChangedBy: bdubbs $
$Date: 2004-08-07 18:56:30 -0600 (Sat, 07 Aug 2004) $

diff -urN unzip-5.50/api.c unzip-5.50-rcl/api.c
--- unzip-5.50/api.c	Thu Nov 22 23:43:26 2001
+++ unzip-5.50-rcl/api.c	Sat Sep 28 14:31:51 2002
@@ -48,6 +48,11 @@
 #endif
 #include "unzvers.h"
 
+/* This is defined as zlibVersion() in zlib.h version 1.1.4 */
+#ifdef   zlib_version
+#  undef zlib_version
+#endif
+
 #ifdef DLL      /* This source file supplies DLL-only interface code. */
 
 jmp_buf dll_error_return;
